package.json

```diff
@@ -472,6 +472,16 @@
           "default": [],
           "markdownDescription": "Extra arguments to pass to ocamllsp."
         },
+        "ocaml.server.codelens": {
```

Comment:
Can't we put all the settings inside the `ocaml.server.settings` object?

Reply:
As for as I could see it's not possible to define an object with a specific set of fields. So if we want those options to be nicely available in the GUI of vscode I think they have to be independent of each other and not in an object. But I'm happy to be corrected if this isn't true.